packages/explorer/src/components/ui/Table.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { cn } from "../../lib/utils";
+import { cn } from "../../app/(explorer)/lib/utils";
```

Comment:
maybe we need to move `cn` out of this dir?

Reply:
sure, move back to the root? I moved all utils used only from within the `explorer` into here but agree that somehow doesn't look right

Reply:
moved it back to root so that common small util functions potentially useful across the whole app are stored in `src/utils.ts`.

Reply:
I generally follow the rule that when you import a thing from somewhere else deep in the app, it's probably time to lift it up into some common dir that can be used in both places.